libraries/OTAUpdate/examples/WiFiFirmwareOTA/WiFiFirmwareOTA.ino

```diff
@@ -0,0 +1,127 @@
+/*
+  OTA
+
+  This sketch demonstrates how to make an self OTA Update the UNO R4 WiFi Firmware
```

Comment:
```suggestion
  This sketch demonstrates how to make the UNO R4 WiFi self-update its Wi-Fi module firmware via OTA
```

The previous wording was incorrect.

My intent was for my suggested text to have the same meaning as what was intended by the former text, but the meaning of the former text was difficult to decipher so it is possible I inadvertently shifted the meaning.